manifests/server/command_processing.pp

```diff
@@ -6,12 +6,6 @@
   $confdir         = $puppetdb::params::confdir,
 ) inherits puppetdb::params {
 
-
-  file { "${confdir}/config.ini":
```

Comment:
This just moved to global.pp, because well, it didn't feel like it should be here? I don't know - maybe we should just collapse both these pp files into one.

There are a lot of refactoring possibilities around configuration I think, right now we just keep copying files ... perhaps we can refactor it all into one big file, seems like its scaffolding for scaffolding sake.